File: Arrays/Duplicate-Values.py
class Solution(object):
    def containsDuplicate(self, nums):
        """
        :type nums: List[int]
        :rtype: bool
        """
        arr = set()
        for i in nums:
            if i not in arr:
                arr.add(i)
            else:
                return True
        return False

File: Arrays/Duplicate-Values.java
class Solution {
    public boolean containsDuplicate(int[] nums) {
        Set<Integer> numbers = new HashSet<Integer>();

        for(int i:nums){
            if(numbers.contains(i)){
                return true;
            }
            numbers.add(i);
        }

        return false;
    }
}